### DISCUSSION TITLE
How to implement diarization using Whisper model

### DISCUSSION THREAD

**Discussion by Arasun01** [openai/whisper#1475] [Q&A]
How do we will find the speaker-level transcripts using whisper. We have a conversation audio file, I would like to find the transcripts per speaker.

**Comment by phineas-pta** [comment]
search for "diarization" u find other people sharing their code

**Comment by Arasun01** [comment]
I was following the thread https://towardsdatascience.com/unlock-the-power-of-audio-data-advanced-transcription-and-diarization-with-whisper-whisperx-and-ed9424307281. It does not work for me with the latest version.

**Reply by phineas-pta** [reply]
ask its author

**Comment by Tex2002ans** [comment]
> How do we will find the speaker-level transcripts using whisper. [...] I would like to find the transcripts per speaker.

A few months ago I answered this in:

- https://github.com/openai/whisper/discussions/1191#discussioncomment-5516405

and linked linked to many of the previous "speaker diarization" topics covering this.

**Reply by Majdoddin** [reply]
@Arasun01 @phineas-pta @Tex2002ans 
[www.lexicaps.com](http://www.lexicaps.com/) seamlessly adds diarization to Whispers transcription. No 3rd party packages.
Announcement: https://github.com/openai/whisper/discussions/1537
Repo: https://github.com/Majdoddin/lexicaps

**Reply by ngtrphuong** [reply]
@Majdoddin - The domain you provided is in-accessible

**Comment by Arasun01** [comment]
Thanks, I will take a look.
